src/com/cdd/bao/axioms/CellLineFix.java

```diff
@@ -0,0 +1,582 @@
+			json.put("cloURI", cell.cloURI);
+			json.put("cloLabel", cell.cloLabel);
+			list.put(json);
+		}
+	
+		try (Writer wtr = new FileWriter("cellpairs.json"))
+		{
+			wtr.write(list.toString(2));
+		}
+	}*/
+	
+	private void flattenBranch(Map<String, String> map, Vocabulary.Branch branch, boolean reqBrenda)
+	{
+		boolean isBrenda = ModelSchema.collapsePrefix(branch.uri).startsWith("obo:BTO_");
+		if (isBrenda == reqBrenda) map.put(branch.uri, branch.label);
+		for (Vocabulary.Branch child : branch.children) flattenBranch(map, child, reqBrenda);
+	}
+
+	private List<Match> matchByLabel(String uri, String label, Map<String, String> map)
+	{
+		Pattern pat = Pattern.compile("(.+)\\s+cell(\\s+line)?\\s*$");
+		Matcher mat1 = pat.matcher(label);
+		String label1 = mat1.matches() ? mat1.group(1) : label;
+
+		List<Match> possibleMatches = new ArrayList<>();
+		for (Map.Entry<String, String> entry : map.entrySet())
+		{
+			Matcher mat2 = pat.matcher(entry.getValue());
+			String label2 = mat2.matches() ? mat2.group(1) : entry.getValue();
+
+			Match m = new Match();
+			m.uri = entry.getKey();
+			m.label = entry.getValue(); // save original label
+			m.score = stringSimilarity(label1, label2);
+
+			possibleMatches.add(m);
+		}
+
+		if (possibleMatches.size() == 0) return new ArrayList<>();
+		possibleMatches.sort((m1, m2) -> m1.score - m2.score);
+
+		// list of actual matches logged 
+		List<Match> actualMatches = new ArrayList<>();
+
+		int bestScore = possibleMatches.get(0).score;
+		if (bestScore <= 2) // only consider matches when the best match has 2 or fewer character tweaks
+		{
+			Util.writeln();
+			int count = 0;
+			for (Match m : possibleMatches)
+			{
+				if (m.score > maxScore) break;
+
+				String uri1 = ModelSchema.collapsePrefix(uri);
+				String uri2 = ModelSchema.collapsePrefix(m.uri);
+
+				JSONObject json = new JSONObject();
+				json.put("uri1", uri1);
+				json.put("label1", label);
+				json.put("uri2", uri2);
+				json.put("label2", m.label);
+				Util.writeln(json.toString() + " / score=" + m.score);
+
+				handleMatchedTerms(uri1, label, uri2, m.label);
+				actualMatches.add(m);
+
+				if (++count >= 10) break;
+			}
+		}
+		return actualMatches;
+	}
+
+	// uri1 and uri2 should already be collapsed
+	private void handleMatchedTerms(String uri1Pfx, String label1, String uri2Pfx, String label2)
+	{
+		String uri1Exp = ModelSchema.expandPrefix(uri1Pfx);
+		String uri2Exp = ModelSchema.expandPrefix(uri2Pfx);
+		String cloDesc = vocab.getDescr(uri1Exp), brendaDesc = vocab.getDescr(uri2Exp);
+		brendaMatched.add(uri2Exp);
+
+		if (!StringUtils.isEmpty(brendaDesc) && !StringUtils.equals(cloDesc, brendaDesc))
+		{
+			String newDesc = !StringUtils.isEmpty(cloDesc) ? (cloDesc + " " + brendaDesc) : brendaDesc;
+			outWriter.println("# incorporate description from BRENDA term into CLO term and then delete BRENDA term");
+			outWriter.println(uri1Pfx + " obo:IAO_0000115 \"" + newDesc + "\" .");
```

Comment:
You will need some basic escaping here... there shouldn't be anything too diabolical, but a basic clean to check for things like quotes and newlines needs to be instituted.